requirements/dev.in

```diff
@@ -1,62 +1,26 @@
 # Dependencies needed only for development/testing.
-colorama==0.4.3
-coverage==5.0.3
-pytest-cov==2.8.1
-pytest==5.3.5
+pytest-cov
 django-debug-toolbar==2.2
-mock==4.0.1
-responses==0.10.9
-django-extensions==2.2.8
-pytest-selenium==1.17.0
-PyPOM==2.2.0
+mock
+responses
+django-extensions
+pytest-selenium
+PyPOM
 
 # Required by django-extension's runserver_plus command.
-Werkzeug==1.0.0
-flake8==3.7.9
-isort==4.3.21
-
-# Required by pluggy
-zipp==3.0.0
-importlib-metadata==1.5.0
-
-# Required by pytest
-atomicwrites==1.3.0
-more-itertools==8.2.0
-packaging==20.1
-pluggy==0.13.1
-py==1.8.1
-pyparsing==2.4.2
-wcwidth==0.1.8
+Werkzeug
+flake8
+isort
 
 # Required by mock
-funcsigs==1.0.2
-pbr==5.4.4
+funcsigs
+pbr
```

Comment:
Not in use anymore.

Reply:
I am concerned others may be using these to debug.  removed